tests/test_collate.py

```diff
@@ -28,12 +28,12 @@
 class test_Collate(ut.TestCase):
     def test_collate_labels(self):
         # Create fake labels
-        labels_size_dict = {
-            "graph_label1": [1],
-            "graph_label2": [3],
-            "node_label2": [5],
-            "edge_label3": [5, 2],
-            "node_label4": [5, 1],
+        labels_num_cols_dict = {
+            "graph_label1": 1,
+            "graph_label2": 3,
+            "node_label2": 1,
+            "edge_label3": 2,
+            "node_label4": 1,
```

Comment:
Aren't node labels and edge labels supposed to be lists? Since there are many nodes / edges?

Reply:
For collation purposes, the lists are always padded to the number of nodes in the graph for node labels, the number of edges in the graph for edge labels, and the number of nodes squared for nodepair labels, and that information is available via `num_nodes` and `num_edges`, so it doesn't need to be specified again here.  This just indicates the number of columns for each task, which doesn't vary between graphs.